package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "hint": "^2.0.0"
+  }
+}
```

Comment:
- [ ] Kindly make sure your app runs correctly on your machine. 👍🏻 Actually I'm still getting this error when I run `npm install` then `npm start` . Please solve this 😄 

![today](https://user-images.githubusercontent.com/79058364/179324933-b40f4707-3006-4a97-8d50-2db267e32c8e.png)